src/util/std_code.h

```diff
@@ -2104,17 +2077,17 @@ class side_effect_expr_function_callt:public side_effect_exprt
   }
 
   side_effect_expr_function_callt(
-    const exprt &_function,
-    const exprt::operandst &_arguments,
-    const typet &_type,
-    const source_locationt &loc)
+    exprt _function,
+    exprt::operandst _arguments,
+    typet _type,
+    source_locationt loc)
     : side_effect_exprt(
         ID_function_call,
-        {_function, exprt(ID_arguments)},
-        _type,
-        loc)
+        {std::move(_function), exprt(ID_arguments)},
+        std::move(_type),
+        std::move(loc))
   {
-    arguments() = _arguments;
+    arguments() = std::move(_arguments);
   }
```

Comment:
You could make the above `multi_ary_exprt(ID_arguments, std::move(_arguments))`

Reply:
Done.